src/components/app/content/cardList/card/index.js

```diff
@@ -33,9 +34,17 @@ function Card(props) {
 
     const editMode = () => setEditable(true);
 
-    let history = useHistory();
-    const cardHandler = () =>
-        editable || history.push({ pathname: `/card/:${cardData.id}`, cardData });
+    const history = useHistory();
+    const cardHandler = () => {
+        const path = `/card/:${cardData.id}`;
```

Comment:
Specifying hard-coded path is a bad idea. If the location changes - component will need to be modified.
As an option you can pass a prop saying if the card is rendered in a card list or on a page. If on list - add handler to the card header otherwise - not assign handler at all.